librz/bin/format/mach0/mach0.c

```diff
@@ -2214,6 +2214,25 @@ static bool __isDataSection(RzBinSection *sect) {
 	return false;
 }
 
+RzList *MACH0_(get_virtual_files)(RzBinFile *bf) {
```

Comment:
Dont you need to check here that `bf` is also not NULL?

Reply:
Doesn't hurt at least.